ingestors/kafka/src/main/scala/hydra/kafka/programs/CreateTopicProgram.scala

```diff
@@ -126,3 +126,7 @@ final class CreateTopicProgram[F[_]: Bracket[*[_], Throwable]: Sleep: Logger](
     } yield ()).use(_ => Bracket[F, Throwable].unit)
   }
 }
+
+object CreateTopicProgram {
```

Comment:
Is there something that was supposed to be placed here that wasn't or can we delete this companion object?

Reply:
Thanks, I was going to put an error there, but then decided to go with the algebra instead. Will remove.